### PR TITLE
yes/no button default value issue is fixed for upload docs

### PR DESCRIPTION
**Pull request by swaroopa86**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/app/case/CosApiClient.ts [hunk 1]
```diff
@@ -124,6 +124,9 @@ export class CosApiClient {
         serviceAuthorization: getServiceAuthToken(),
       };
 
+      console.log('Auth token: {}', user.accessToken);
+      console.log('Service auth token: {}', getServiceAuthToken());
+
```
Comment (AlokDatta) [inline src/main/app/case/CosApiClient.ts:129]: remove this

File: src/main/app/controller/GetController.ts [hunk 2]
```diff
@@ -61,7 +61,8 @@ export class GetController {
     });
   }
 
-  private getPreferredLanguage(req: AppRequest) {
+  // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
```
Comment (AlokDatta) [inline src/main/app/controller/GetController.ts:64]: why disabling next line check?

File: src/main/app/controller/PostController.ts [hunk 3]
```diff
@@ -40,7 +39,7 @@ export class PostController<T extends AnyObject> {
     } else if (req.body.onlyContinue) {
       await this.onlyContinue(req, res, form, formData);
     } else {
-      await this.getCaseList(req, res, form, formData);
+      //await this.getCaseList(req, res, form, formData);
```
Comment (AlokDatta) [inline src/main/app/controller/PostController.ts:42]: remove unwanted code

**Review by AlokDatta — CHANGES_REQUESTED** [review]

File: src/main/app/controller/PostController.ts (hunk 3 above)
Comment (swaroopa86) [reply src/main/app/controller/PostController.ts:42]: remove this commented line

File: src/main/app/case/CosApiClient.ts (hunk 1 above)
Comment (swaroopa86) [reply src/main/app/case/CosApiClient.ts:129]: removed this console.log info

**Pull request closed without merging**